Module/src/PSWordCloud/NewWordCloudCommand.cs

```diff
@@ -809,6 +808,53 @@ protected override void EndProcessing()
 
         #region HelperMethods
 
+        private IEnumerable<string> NormalizeInput(PSObject input)
+        {
+            if (MyInvocation.ExpectingInput)
+            {
+                if (input.BaseObject is string s)
+                {
+                    yield return s;
+                }
+                else
+                {
+                    yield return LanguagePrimitives.ConvertTo<string>(input);
+                }
+            }
+            else
+            {
+                switch (input.BaseObject)
+                {
+                    case string[] sa:
+                        foreach (var line in sa)
+                        {
+                            yield return line;
+                        }
+
+                        yield break;
+
+                    case string s2:
+                        yield return s2;
+                        yield break;
+
+                    default:
+                        var enumerable = LanguagePrimitives.GetEnumerable(input.BaseObject);
+                        if (enumerable != null)
+                        {
+                            foreach (var item in enumerable)
+                            {
+                                yield return LanguagePrimitives.ConvertTo<string>(item);
```

Comment:
`ConvertTo<string>` can throw if the implementer throws in their `ToString` method.  Dunno if you want to surface those exceptions.

Reply:
That's so _cursed_ but I **guess** it might be okay to silence those.